### PR TITLE
Fix Project

### PR DESCRIPTION
**Pull request by sondth-1037**
## Related Tickets:
- N/A
## WHAT this PR does?
- Fix Routing name, js for Agency
## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by sondth-1037** [comment]
ready

**Comment by sondth-1037** [comment]
ready

File: app/Http/Controllers/Agency/AgencyController.php [hunk 1]
```diff
@@ -137,6 +137,6 @@ public function postLogout(Request $request)
         Auth::logout();
         alert()->success(trans('contents.common.alert.title.logout_success'), trans('contents.common.alert.message.logout_success'));
         
-        return  redirect()->route('agency.getLogin');
+        return  redirect()->route('agency.agency.getLogin');
```
Comment (hoangminhuet) [inline app/Http/Controllers/Agency/AgencyController.php]:
```suggestion
        return redirect()->route('agency.agency.getLogin');
```

File: resources/views/agency/createRequest/index.blade.php [hunk 2]
```diff
@@ -12,9 +12,9 @@
             <div class="col-sm-6">
                 <ol class="breadcrumb float-sm-right">
                     <li class="breadcrumb-item"><a
-                            href="{{ route('requests.create') }}">{{ trans('contents.common.home') }}</a></li>
+                            href="{{ route('agency.requests.create') }}">{{ trans('contents.common.home') }}</a></li>
```
Comment (hoangminhuet) [inline resources/views/agency/createRequest/index.blade.php]: indent 4 spaces

File: resources/views/agency/createRequest/index.blade.php [hunk 3]
```diff
@@ -12,9 +12,9 @@
...
                     <li class="breadcrumb-item"><a
-                            href="{{ route('requests.index') }}">{{ trans('contents.common.request') }}</a></li>
+                            href="{{ route('agency.requests.index') }}">{{ trans('contents.common.request') }}</a></li>
```
Comment (hoangminhuet) [inline resources/views/agency/createRequest/index.blade.php]: tương tự

File: resources/js/agency.js [hunk 4]
```diff
@@ -317,7 +317,7 @@ $('#btn-from-airport-submit').click(function(e)
     var flight_no = $('#from-airport-flight').val()
     var note;
     if (flight_no) {
-        note = "<li>Fligt no: " + flight_no + "</li> ";
+        note = "Flight no: " + flight_no + "\n";
```
Comment (hoangminhuet) [inline resources/js/agency.js]: dùng Interpolate String ES6 anh nhé

File: app/Http/Controllers/Agency/AgencyController.php (hunk 1 above)
Comment (sondth-1037) [reply app/Http/Controllers/Agency/AgencyController.php]: oke em

File: resources/views/agency/createRequest/index.blade.php (hunk 2 above)
Comment (sondth-1037) [reply resources/views/agency/createRequest/index.blade.php]: oke em

File: resources/views/agency/createRequest/index.blade.php (hunk 3 above)
Comment (sondth-1037) [reply resources/views/agency/createRequest/index.blade.php]: oke em

File: resources/js/agency.js (hunk 4 above)
Comment (sondth-1037) [reply resources/js/agency.js]: oke em :+1:

**Pull request merged**